### ISSUE TITLE
line after table is removed after formatting

### ISSUE THREAD

**Issue by tessus** [roman-r-m/joplin-plugin-table-formatter#3]
Version 1.1.0

![tfi1](https://user-images.githubusercontent.com/223439/111539513-59db4400-8744-11eb-855a-7d928eeebb74.gif)

```
Joplin 1.8.0 (dev, darwin)

Client ID: e354a932e6e143c782264aad4112b674
Sync Version: 2
Profile Version: 34
Keychain Supported: Yes

Revision: f3b2ca475 (dev)
```

**Comment by roman-r-m** [comment]
Thanks for reporting, fixed in version 1.1.1 which I've just published

**Comment by tessus** [comment]
Thanks.
